index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ghazanfar Portfolio</title>
+  </head>
+  <body>
+    <div class="main-container">
+      <section class="toolbar">
+        <div class="logo">
+          <p>My Logo</p>
+        </div>
+
+        <div class="icon">
+          <img src="./images/Union.png" alt="Menu Icon" />
+        </div>
```

Comment:
- [ ] Hi. Please as part of the requirements please kindly wrap the menu icon in a nav tag